diagnostic.md

```diff
@@ -35,7 +39,9 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    1.)In the first route, the product folder is nested inside of the products folder.
+    2.)The paths for when the product route ('product') is accessed are
```

Comment:
Actually, the urls here would be the same because of nesting. 
The first `'product'` route definition is nested under the `'products'`
definition, whereas the second `'product'` route definition is a sibling of
the `'products'` definition.

Nested route definitions, like the first `'product'` route above, require
a nested file structure, such that Route and Template files for `'product'`
are expected to be found at `app/products/product`. For sibling routes,
folders are siblings as well.

The first `'product'` route definition instructs Ember to render
`app/products/product/template.hbs` in the context (the `outlet`) of the
parent `app/products/template.hbs`. The second `'product'` route definition
instructs Ember to render `app/product/template.hbs` in the context of the
application template.

These two route definitions do **not** have different paths.

.5/1